nc_test/Makefile.am

```diff
@@ -25,21 +25,28 @@ TEST_EXTENSIONS = .sh
 check_PROGRAMS =
 
 # These are the tests which are always run.
-TESTPROGRAMS = t_nc tst_small nc_test tst_misc tst_norm tst_names	\
-tst_nofill tst_nofill2 tst_nofill3 tst_atts3 tst_meta tst_inq_type	\
```

Comment:
I'm curious about the removal of some of these test programs and/or moving them into the pnetcdf-specific fenceposts.

Reply:
Good catch @WardF!

Indeed, these programs can be run both with and without PNETCDF. They contain code like this:

```
#ifdef USE_PNETCDF
	ret = nc_open_par(fname,omode, MPI_COMM_WORLD, MPI_INFO_NULL, &id);
#else
	ret = nc__open(fname,omode, &chunksz, &id);
#endif

```

So these programs are intended to always be built, not just with PNETCDF.

Reply:
In that case they could be unconditionally added to check_PROGRAMS. Someone would then need to figure out the best way to run them in both cases.